src/renderer/components/ConfirmationModal.tsx

```diff
@@ -0,0 +1,55 @@
+import styled from "@emotion/styled";
+import Button from "@material-ui/core/Button";
+import Dialog from "@material-ui/core/Dialog";
+import DialogActions from "@material-ui/core/DialogActions";
+import DialogContent from "@material-ui/core/DialogContent";
+import DialogTitle from "@material-ui/core/DialogTitle";
+import { useTheme } from "@material-ui/core/styles";
+import useMediaQuery from "@material-ui/core/useMediaQuery";
+import React from "react";
+
+export interface ConfirmationModalProps {
+  open: boolean;
+  onClose: () => void;
+  onSubmit: () => void;
+  title: string;
+  content: string;
+}
+
+export const ConfirmationModal: React.FC<ConfirmationModalProps> = ({ open, onClose, onSubmit, title, content }) => {
+  const theme = useTheme();
+  const fullScreen = useMediaQuery(theme.breakpoints.down("xs"));
+
+  const handleSubmit = (e: React.FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+    console.log("submitting form...");
+    onSubmit();
+    onClose();
+  };
+
+  return (
+    <Dialog open={open} onClose={onClose} fullWidth={true} fullScreen={fullScreen} disableBackdropClick={true}>
+      <form onSubmit={handleSubmit}>
+        <StyledDialogTitle>{title}</StyledDialogTitle>
+        <DialogContent style={{ display: "flex" }}>
+          <div>{content}</div>
```

Comment:
```suggestion
          <div>{content}</div>
          {children}
```
I think just remove the `content` prop and allow for children to be rendered.